File: binance4j-blvt/src/main/java/com/binance4j/blvt/param/RedemptionParams.java
package com.binance4j.blvt.param;

import com.binance4j.blvt.client.BLVTClient;
import com.binance4j.core.annotation.Param;
import com.binance4j.core.param.Params;

/**
 * {@link BLVTClient#redeem} params.
 * 
 * @param tokenName Token name.
 * @param amount    Amount to redeem.
 */
@Param
public record RedemptionParams(String tokenName, String amount) implements Params {
}

File: binance4j-blvt/src/main/kotlin/com/binance4j/blvt/param/RedemptionParams.kt
package com.binance4j.blvt.param

import com.binance4j.blvt.client.BLVTClient
import com.binance4j.core.annotation.Param
import com.binance4j.core.param.Params

/**
 * [BLVTClient.redeem] params.
 *
 * @property tokenName Token name.
 * @property amount    Amount to redeem.
 */
@Param
data class RedemptionParams(var tokenName: String, var amount: String) : Params